sklearn/utils/tests/test_validation.py

```diff
@@ -362,3 +364,7 @@ def test_check_consistent_length():
     assert_raises_regexp(TypeError, 'estimator', check_consistent_length,
                          [1, 2], RandomForestRegressor())
     # XXX: We should have a test with a string, but what is correct behaviour?
```

Comment:
argh, I knew I had done something weird: I wrote the same thing into the commit :) https://github.com/ibayer/scikit-learn/commit/30c72f261be6b16f82809c002cdcb6994257c505#commitcomment-10644138